### PR TITLE
Add support for CMCC RAX3000M NAND ubootmod

### PR DESCRIPTION
**Pull request by pengutw**
Add support for CMCC RAX3000M NAND version in custom U-Boot.

### CODE REVIEW

**Comment by pengutw** [comment]
This is for hardware 1027 version

File: target/linux/mediatek/dts/mt7981b-cmcc-rax3000m-nand-ubootmod.dts [hunk 1]
```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+#include "mt7981b-cmcc-rax3000m.dts"
+
+/ {
+	model = "CMCC RAX3000M NAND version (custom U-Boot layout)";
```
Comment (csharper2005) [inline target/linux/mediatek/dts/mt7981b-cmcc-rax3000m-nand-ubootmod.dts:7]: `(custom U-Boot layout)` - this does not apply to the device model.

File: target/linux/mediatek/image/filogic.mk [hunk 2]
```diff
@@ -322,6 +322,29 @@ define Device/cmcc_rax3000m
 endef
 TARGET_DEVICES += cmcc_rax3000m
 
+define Device/cmcc_rax3000m-nand-ubootmod
+  DEVICE_VENDOR := CMCC
+  DEVICE_MODEL := RAX3000M NAND version
```
Comment (csharper2005) [inline target/linux/mediatek/image/filogic.mk:327]: `NAND version` - this does not apply to the device model.

**Review by csharper2005 — COMMENTED** [review]
Why do you need another yet device version entity?

**Comment by pengutw** [comment]
> Why do you need another yet device version entity?

The current production date is 20231027. If you use the previous uboot.

File: target/linux/mediatek/dts/mt7981b-cmcc-rax3000m-nand-ubootmod.dts (hunk 1 above)
Comment (pengutw) [reply target/linux/mediatek/dts/mt7981b-cmcc-rax3000m-nand-ubootmod.dts:7]: ex.CMCC RAX3000M NAND 1027 version

File: target/linux/mediatek/image/filogic.mk (hunk 2 above)
Comment (pengutw) [reply target/linux/mediatek/image/filogic.mk:327]: Do you have any suggestions for device models?
Not sure about the rules here

**Comment by 1715173329** [comment]
I don't think there's any major hardware change in this "newer version". The only update worth to mention, is the manufactory encrypted the backup configuration.

To boot with official OpenWrt firmware, you should flash official bl2 and fip which have NMBM disabled.

**Comment by 981213** [comment]
We don't accept firmware support for non-OpenWrt third-party u-boot.
Closed.

**Pull request closed without merging**